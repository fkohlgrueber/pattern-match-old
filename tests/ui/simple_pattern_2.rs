
fn main() {
    1;

    2;

    100;

    101;

    102;
}
