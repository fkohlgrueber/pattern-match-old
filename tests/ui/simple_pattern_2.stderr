error: This is a match for a simple pattern (2). Well Done!
 --> $DIR/simple_pattern_2.rs:7:5
  |
7 |     100;
  |     ^^^
  |
  = note: #[forbid(simple_pattern_2)] on by default

error: This is a match for a simple pattern (2). Well Done!
 --> $DIR/simple_pattern_2.rs:9:5
  |
9 |     101;
  |     ^^^

error: aborting due to 2 previous errors

